sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_hello_world_async.py

```diff
@@ -23,31 +23,26 @@
 from dotenv import find_dotenv, load_dotenv
 import os
 
+from azure.containerregistry.aio import ContainerRegistryClient
+from azure.identity.aio import DefaultAzureCredential
 
-class CreateClients(object):
+
+class CreateClientsAsync(object):
     def __init__(self):
         load_dotenv(find_dotenv())
+        self.account_url = os.environ["CONTAINERREGISTRY_ENDPOINT"]
```

Comment:
One of the advantages of having account url in the `# [START ...]` block is it will show up in the sample documentation. I think it's worth leaving the account url read where it is so customers understand how they might find this data, or at least what it might look like to get it